MAD.Procore.StudioHoursUpload/Jobs/StudioHourUploadLogProducer.cs

```diff
@@ -32,7 +31,12 @@ public async Task ProduceStudioHoursUploadLogs()
                 .Where(y => y.Region == this.procoreConfig.Name)
                 .ToListAsync();
 
-            var studioHours = await this.studioHourClient.GetStudioHours();
+            //Retrieve last processed studio log and pass processed date time to GetStudioHours method
+            var lastStudioLog = await this.studioHourDbContext.StudioHourUploadLog
+                .OrderByDescending(x => x.ProcessedDate)
+                .FirstOrDefaultAsync(x => x.Region == this.procoreConfig.Name && x.ProcessedDate.HasValue && string.IsNullOrWhiteSpace(x.Error));
+
+            var studioHours = await this.studioHourClient.GetStudioHours(lastStudioLog?.ProcessedDate);
             var today = this.GetTodayUtc();
```

Comment:
This will still only generate a log for today (GetTodayUtc)

It should loop from LastStudioLog's date to Today and generate the missing logs for all the studio hours. GetStudioHours will have to be called for each day in between LastStudioLog and Today, otherwise the office hours & man count will not reflect that particular day